elide-async/src/main/java/com/yahoo/elide/async/export/TableExporter.java

```diff
@@ -99,4 +94,16 @@ public TableExporter(Elide elide, String apiVersion, User user, GraphQLParser gr
 
         return results;
     }
+
+    public TableExportParser getParser(TableExport query) {
```

Comment:
Maybe instead of returning a parser, this should return a load function which returns an Observable of records.